lib/rom/relation.rb

```diff
@@ -189,6 +191,11 @@ def repository
       self.class.repository
     end
 
+    # @api public
+    def to_lazy(*args)
+      Lazy.new(self, *args)
+    end
```

Comment:
Ah,  okay, makes sense ... and since lazy is default return, user really isn't expected to call that.  And a relation is _not_ an Enumerable, I see.  Seems fair.  :)